prime-factors/example.lua

```diff
@@ -1,4 +1,4 @@
-function PrimeFactors(input)
+local function PrimeFactors(input)
```

Comment:
This defined a global that the tests referenced directly. Instead, declare this as local so that only the exported function is made available to the test file.